samples/host_exerciser/host_exerciser_cmd.h

```diff
@@ -165,7 +165,7 @@ class host_exerciser_cmd : public test_command
 
 
         // Number of cache lines
-        d_afu->write64(HE_NUM_LINES, LPBK1_BUFFER_SIZE / (1 * CL));
+        d_afu->write64(HE_NUM_LINES, (LPBK1_BUFFER_SIZE / (1 * CL)) -1);
```

Comment:
Just curious, why the - 1 ?  Is the buffer size not a multiple of cachelines?

Reply:
Specifies the number of cache lines to read and/or write. Actual value is NumCacheLines + 1. 
Ex. 10'h0 - 1 Cache Lines
Ex. 10'h1 - 2 Cache Lines
Ex. 10'h3FF - 1024 Cache Lines

This threshold may be different for each test. Ensure that source and destination buffers are large enough to accommodate NumCacheLines.

N (MAX_DATA_SIZE) is a compilation time parameter and represents the maximum number of cache lines HE can transfer. Default value of N is 1024. This field must be a multiple of CFG.ReqLen. Width is [log2(N)-1:0].